web_ui/frontend/app/api/docs/pelican-swagger.yaml

```diff
@@ -652,7 +743,7 @@ paths:
         For user with admin previlege, they can update any valid namespace.
 
 
-        For general users, they can only update the namespace belonging to the user, or it returns 404
+        For non-admin users, they can only update the namespace belonging to the user, or it returns 404. They also cannot update a namespace if its `admin_metadata.status == approved`.
```

Comment:
Non-admin users can update only namespaces they own and only if the approval status is `admin_metadata.status == approved`, otherwise the endpoint returns 404.